docs/docsite/rst/dev_guidelines.rst

```diff
@@ -480,20 +480,128 @@ When you make a call using boto3, you will probably get back some useful informa
 should return in the module.  As well as information related to the call itself, you will also have
 some response metadata.  It is OK to return this to the user as well as they may find it useful.
 
-Boto3 returns all values CamelCased.  Ansible follows Python standards for variable names and uses
-snake_case. There is a helper function in module_utils/ec2.py called ``camel_dict_to_snake_dict``
+Boto3 commonly returns all keys CamelCased.  Ansible follows Python standards for variable names and uses
+snake_case.  There is a commonly used helper function ``ansible.module_utils.common.dict_transformations.camel_dict_to_snake_dict``
 that allows you to easily convert the boto3 response to snake_case.
 
 You should use this helper function and avoid changing the names of values returned by Boto3.
 E.g. if boto3 returns a value called 'SecretAccessKey' do not change it to 'AccessKey'.
 
+There is an optional parameter, ``ignore_list``, which is used to avoid converting a sub-tree
+of a dict.  This is particularly useful for tags, where keys are case-sensitive.
+
+.. code-block:: python
+
+   # Make a call to AWS
+   result = connection.aws_call()
+
+   # Return the result to the user without modifying tags
+   module.exit_json(changed=True, **camel_dict_to_snake_dict(result, ignore_list=['Tags']))
```

Comment:
For consistency with _info modules, this might be better as `some_resource=camel_dict_to_snake_dict(result, ignore_list=['Tags'])`

Reply:
Updated!